Update support articles to include invoice filtering for enterprise customers (#1463)
dnsimple/dnsimple-support: content/articles/account-invoice-history.md

1. On the dashboard, choose the account you want to see the invoice history for.

1. Click **Account** at the top right, then click the **Billings and plans** tab on the left side.

1. Scroll down to the <label>Invoices</label> section and click on <label>View all</label>. You can also view your last invoice directly by clicking **View** in the **Last Invoice** section.

    ![Invoice History Link](/files/account-billing-view-invoice-history-link.png)
</div>

<info>
Only available for Enterprise clients.
</info>

[Enterprise](https://dnsimple.com/sales) customers need quick, reliable access to invoices for auditing, reconciliation, and compliance. Our new filters make it easy to locate specific invoices by ID or content, helping teams save time and streamline financial workflows.

![Enterprise Invoice Filter](/files/enterprise-invoice-filters.png)

## Payment states

Your invoices may be in certain states depending on different circumstances:

### `finalized` {#invoice-state-finalized}

> An invoice marked as `finalized` has been generated and is ready for payment, but no collection attempts have been made yet. Because we process invoices immediately, you'll rarely see invoices linger in this state—they almost always move on to either `collected` or `failed` within minutes.

Author: @jacegu would you say this is the right definition?

Author: Kind bump @jacegu

Reviewer: ```suggestion
A `finalized` invoice is ready for payment, but no collection attempts have been made yet. Because we usually process invoices immediately, you'll rarely see invoices linger in this state as they almost always move on to either `collected` or `failed` within minutes.
```

Author: Thx!
